website/README.md

```diff
@@ -55,10 +56,10 @@ Before opening a pull request or push to git repo, you can preview changes from
 
 ## Pushing to site (for committer only)  
 1. `cd website/_site`
-2. `svn status`
-3. You'll need to `svn add` any new files
-4. `svn commit -m 'UPDATE MESSAGE'`
+2. `git status`
+3. You'll need to `git add` any new files
+4. `git commit -m 'UPDATE MESSAGE'`
 
-Within a few minutes, svnpubsub should kick in and you'll be able to
+Within a few minutes, gitpubsub should kick in and you'll be able to
 see the results at
-[http://kylin.apache.org](http://kylin.apache.org/).
\ No newline at end of file
+[https://kylin.apache.org](https://kylin.apache.org/).
```

Comment:
Hi @martin-g. The apache kylin official website protocol is indeed `http` rather than `https`.